test/io/fixtures.yaml

```diff
@@ -153,6 +153,41 @@ roleclaims:
       role: postgrest_test_author
       other: true
     expected_status: 401
+  - key: '.realm_access.roles[?(@ == "postgrest_test_author")]'
+    data:
+      realm_access:
+        roles:
+          - other
+          - postgrest_test_author
+    expected_status: 200
+  - key: '.realm_access.roles[?(@ != "other")]'
+    data:
+      realm_access:
+        roles:
+          - other
+          - postgrest_test_author
+    expected_status: 200
+  - key: '.realm_access.roles[?(@ ^== "postgrest_te")]'
```

Comment:
Q: Where is documented that `@` means the left value? It does look good, but would like to know.

Reply:
It comes from the original source of JSONPath [https://goessner.net/articles/JsonPath/](https://goessner.net/articles/JsonPath/) and [https://github.com/JSONPath-Plus/JSONPath](https://github.com/JSONPath-Plus/JSONPath). We can also document this in our docs.